crates/ruff_linter/src/rules/pydocstyle/rules/not_missing.rs

```diff
@@ -10,6 +10,8 @@ use ruff_text_size::TextRange;
 use crate::checkers::ast::Checker;
 use crate::registry::Rule;
 
+/// This rule is ignored for Jupyter Notebooks.
```

Comment:
Should this be under the *What it does section*?

Reply:
Makes sense. I think it's a bit too prominent. We could introduce a new `Limitations` section maybe also with an explanation why the rule is ignored.

Reply:
I've added a new `Notebook behavior` section to several rules for when they have different Jupyter notebook behavior.